semgrep-core/src/core/ast/AST_generic.ml

```diff
@@ -1749,8 +1749,7 @@ and directive_kind =
   | ImportFrom of
       tok (* 'import'/'from' for Python *)
       * module_name
-      * ident
-      * alias option (* as name alias *)
+      * (ident * alias option (* as name alias *)) list
```

Comment:
Could we add a comment here explaining why we don't "desugar" in multiple ImportFrom like
we do for variable declarations. I already forgot why we do that ...

Reply:
https://github.com/returntocorp/semgrep/pull/6634